syntax = "proto3";

message LocationEPMessage {
  int32 person_id = 1;
  string longitude = 2;
  string latitude = 3;
  string creation_time = 4;
}

service LocationEPService {
  rpc Create(LocationEPMessage) returns (LocationEPMessage);
}
